### ISSUE TITLE
Imports in current project are not renamed

### ISSUE THREAD

**Issue by Frozenlock** [benedekfazekas/mranderson#92]
When using `^:inline-dep` with a dependency, imports inside this dependency are renamed as expected.

However, imports used in the current project but depending on an inlined dependency are not renamed.

## To reproduce

See this fork as an example:
https://github.com/Frozenlock/congomongo

```
lein do clean, inline-deps
```
Now check `target/srcdeps/somnium/congomongo.clj`
Imports from the inlined dependency should be renamed, but aren't.
(com.mongodb.client, org.bson.types, etc.)

**Comment by lread** [comment]
@Frozenlock, I gave your repo a whirl and confirm the imports are not being renamed.

**Comment by lread** [comment]
@FrozenLock, on a whim, I tried adding clj-yaml as an inlined-dep to your congomongo project.

Clj-yaml is a light wrapper for the SnakeYAML java library.
 
After running `lein do clean, inline-deps` on congomongo, I see that an inlined clj-yaml is now referencing an inlined SnakeYAML. 

So maybe inlined a java lib only currently only works from from a secondary dep.

@benedekfazekas might know more than me on this one. 

One thing to keep in mind is that inlining is not always perfect and should only be used if you absolutely need it. Do you absolutely need to inline in congomongo?

**Comment by Frozenlock** [comment]
No... when I realized that mranderson wasn't working, I decided to bite the bullet and refactor the whole codebase at once.

I opened the ticket because it looked like a bug.

**Comment by lread** [comment]
Thanks for the issue @Frozenlock. 

If any lurkers out there need this to work, pipe up, and maybe we'll find some time to lend a hand.
